packages/sdk-js/src/GrowthBook.ts

```diff
@@ -147,7 +189,7 @@ export class GrowthBook<
     await refreshFeatures(
       this,
       options.timeout,
-      options.skipCache || this._ctx.enableDevMode,
+      options.skipCache || this._ctx.enableDevMode || this._ctx.remoteEval,
```

Comment:
Forcing it to always skip cache seems like overkill. We lose a lot of the benefits of having a cache in the first place. I think we might want to selectively enable this for certain changes like when  `setAttributes` is called